docker/safety-rules/build.sh

```diff
@@ -5,4 +5,4 @@ set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
-$DIR/../libra-build.sh $DIR/Dockerfile libra_safety_rules "$@"
+$DIR/../libra-build.sh $DIR/Dockerfile libra/safety_rules "$@"
```

Comment:
now this should be libra/validator_tcb

Reply:
Is the intent that we'll have a single docker container containing all the varieties of tcb? (key-manager, safety-rules, execution -- eventually)